libvcx/src/api/disclosed_proof.rs

```diff
@@ -401,6 +401,44 @@ pub extern fn vcx_disclosed_proof_get_requests(command_handle: CommandHandle,
     error::SUCCESS.code_num
 }
 
+#[no_mangle]
+pub extern fn vcx_v2_disclosed_proof_get_requests(command_handle: CommandHandle,
```

Comment:
As I mentioned, I am not much in favor of adding this unless we have usecase as extending api is adding up future maintenance cost. I'll leave the decision up to your judgement.

Reply:
Resolved in another comment.